src/components/badge/Badge.tsx

```diff
@@ -13,28 +13,29 @@ import {
   IOBadgeVSpacing,
   IOColors,
   useIOExperimentalDesign,
-  useIOTheme,
   useIOThemeContext
 } from "../../core";
 import { useIOFontDynamicScale } from "../../utils/accessibility";
 import { WithTestID } from "../../utils/types";
 import { IOText } from "../typography";
 
+/* 
+NOTE FOR REFACTORING in the `io-app`:
+
+- default (legacy) -> removed
+- blue -> Deleted -> replaced with `default`
+- info -> removed -> highlight?
+- purple -> cgn
+- turquoise -> highlight
+- contrast -> removed
+- lightBlue -> default
+*/
+
 export type Badge = WithTestID<{
   outline?: boolean;
   text: string;
   allowFontScaling?: boolean;
-  variant:
-    | "default"
-    | "info"
-    | "warning"
-    | "error"
-    | "success"
-    | "purple"
-    | "lightBlue"
-    | "blue"
-    | "turquoise"
-    | "contrast";
+  variant: "default" | "warning" | "error" | "success" | "cgn" | "highlight";
```

Comment:
Since the `variant` parameter is required, what do you think to initialize its value to `default`?

Reply:
@LeleDallas The `Badge' component always gives a meaning when used, even when it's set as `default'. I don't remember exactly why I set the props as required, but it probably had to do with the side effect when the props are not set and the component uses the fallback. If we change the fallback, some parts of the application will show a different badge, thus conveying a different meaning. Maybe I'm being too pedantic here, but I prefer to keep it mandatory.

Another reason is that most of these badges are configured through a config object, and it's better to stay strict at the TypeScript level.